docs/source/reference/interoperability.rst

```diff
@@ -18,6 +19,10 @@ This enables NumPy ufuncs to be directly operated on CuPy arrays.
     result = numpy.sum(arr)
     print(type(result))  # => <class 'cupy.core.core.ndarray'>
 
+:class:`cupy.ndarray` also implements ``__array_function__`` interface (see `NEP 18 — A dispatch mechanism for NumPy’s high level array functions <http://www.numpy.org/neps/nep-0018-array-function-protocol.html>`_ for details).
+This enables code using NumPy to be directly operated on CuPy arrays.
+``__array_function__`` feature requires NumPy 1.16 or later; note that this is currently defined as an experimental feature of NumPy and you need to specify the environment variable to enable it.
```

Comment:
Why don't you mention the name of the environment variable (`NUMPY_EXPERIMENTAL_ARRAY_FUNCTION`)?  It'll be clearer that the variable is read by NumPy.